<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="KylesPC" name="com.revature.exceptions.InvalidRequestExceptionTest" tests="4" failures="0" timestamp="16 Oct 2019 19:36:38 GMT" time="0.015" errors="0">
  <testcase name="InvalidRequestExceptionMessageInjectedBySuperTestFalse" time="0.004" classname="com.revature.exceptions.InvalidRequestExceptionTest"/>
  <testcase name="InvalidRequestExceptionMessageInjectedByThrowTestFalse" time="0.004" classname="com.revature.exceptions.InvalidRequestExceptionTest"/>
  <testcase name="InvalidRequestExceptionMessageInjectedByThrowTestTrue" time="0.004" classname="com.revature.exceptions.InvalidRequestExceptionTest"/>
  <testcase name="InvalidRequestExceptionMessageInjectedBySuperTestTrue" time="0.003" classname="com.revature.exceptions.InvalidRequestExceptionTest"/>
</testsuite> <!-- com.revature.exceptions.InvalidRequestExceptionTest -->
